common/src/main/java/org/opensearch/ml/common/MLModelGroup.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.commons.authuser.User;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
+
+@Getter
+public class MLModelGroup implements ToXContentObject {
+    public static final String MODEL_GROUP_NAME_FIELD = "name"; //name of the model group
+    // We use int type for version in first release 1.3. In 2.4, we changed to
```

Comment:
Seems this comment is not for model group, model group is something new in 2.8, not released in 1.3

Reply:
Removed it